<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDiaporama</class>
 <widget class="QDialog" name="DialogDiaporama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lchangerDiapo">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Veuillez selectionner le diaporama que vous voulez charger :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbDiapoPantxika">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>diapoama Pantxika</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDiapoThierry">
       <property name="text">
        <string>diaporama Thierry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDiapoYann">
       <property name="text">
        <string>diaporama Yann</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDiapoManu">
       <property name="text">
        <string>diaporam Manu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
